### PR TITLE
fix(App): load county and status after signUp

### PR DESCRIPTION
**Pull request by ItayBenMoshe**
[582](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/recentlyupdated/)

### CODE REVIEW

**Review by MayaSasson — DISMISSED** [review]

File: client/src/components/App/App.tsx [hunk 1]
```diff
@@ -37,14 +37,16 @@ const App: React.FC = (): JSX.Element => {
     const [isUserUpdated, setIsUserUpdated] = useState<boolean>(true);
     const [isSignUpOpen, setIsSignUpOpen] = useState<boolean>(false);
 
+    const handleOpenSignUp = () => setIsSignUpOpen(true);
```
Comment (NaamanTsur) [inline client/src/components/App/App.tsx:40]: I don't think you are gaining much from wrapping the `setIsSignUpOpen` with handle functions

**Review by NaamanTsur — CHANGES_REQUESTED** [review]

File: client/src/components/App/App.tsx (hunk 1 above)
Comment (ItayBenMoshe) [reply client/src/components/App/App.tsx:40]: just to make it understandable

Comment (NaamanTsur) [reply client/src/components/App/App.tsx:40]: I don't think `handleOpenSignUp` adds more readability than `setIsSignUpOpen(true)` in this scenario.

Comment (ItayBenMoshe) [reply client/src/components/App/App.tsx:40]: fixed

**Review by NaamanTsur — APPROVED** [review]

**Review by MayaSasson — APPROVED** [review]

**Pull request merged**